server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CICSLexer.g4

```diff
@@ -13,6 +13,7 @@
  */
 
 lexer grammar CICSLexer;
+channels{COMMENTS}
```

Comment:
Had to add the channel to each lexer, as it seems the channel doesn't work for combined grammar.

ref - https://github.com/antlr/antlr4/issues/1555
ref - https://github.com/antlr/antlr4/issues/965